test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.cs

```diff
@@ -76,7 +76,7 @@ public void CannotStartConnectionWithInvalidTransportType(TransportType requeste
         }
 
         [Fact]
-        public async Task CannotStartStoppedConnection()
+        public async Task CannotStartConnectionDisposedAfterStarting()
```

Comment:
`CannotStartDisposedConnection`

Reply:
A test called `CannotStartDisposedConnection` already exists and this one tests a little different scenario.